.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,7 +18,7 @@ Learn more about contributing: [CONTRIBUTING.md](https://github.com/nf-core/sare
 - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
 - [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
 - [ ] Make sure your code lints (`nf-core lint`).
-- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
+- [ ] Ensure the test suite passes (`nf-test test main.nf.test -profile test,docker`).
```

Comment:
Where is one supposed to run that command?  In the project root dir?

Reply:
good point, that's not well done

Reply:
and the command won't work

Reply:
```suggestion
- [ ] Ensure the test suite passes (`nf-test test tests/ --profile +docker`).
```

Reply:
cf https://github.com/nf-core/tools/pull/2821